packages/react/src/components/auth/AuthBoundary.tsx

```diff
@@ -145,6 +151,7 @@ function AuthSwitch({
 
   const isLoggedOut = authState.type === AuthStateType.LOGGED_OUT && !authState.isDestroyingSession
   const loginUrl = useLoginUrl()
+  useComlinkTokenRefresh() // Initialize the comlink token refresh context for Dashboard Comlink
```

Comment:
i see that you're calling this hook but this hook implementation doesn't seem to have any side effects.it looks like it just gives access to some functions from the context. i think the provider is the one doing the side effects and this line may not be necessary.